pkg/cache/application.go

```diff
@@ -355,7 +348,10 @@ func (app *Application) handleCompleteApplicationEvent(event *fsm.Event) {
 
 func (app *Application) enterState(event *fsm.Event) {
 	log.Logger.Info("app state",
+		zap.String("currentState", event.Src),
+		zap.String("targetState", event.Dst),
+		zap.String("event", event.Event),
 		zap.String("appID", app.applicationID),
 		zap.String("queue", app.queue),
-		zap.String("state", app.GetApplicationState()))
+		zap.String("currentAppState", app.GetApplicationState()))
```

Comment:
This is a double up: we have current and target state already (currentAppState)
Can we also bring this inline with the way this is done in the core? Log at a debug level and with minimal info and the same order: appID, source, destination, and event.

Reply:
Make sense. Align with how the state transition is logged in core.